pandas/io/json/_json.py

```diff
@@ -31,20 +32,23 @@
 
 TABLE_SCHEMA_VERSION = "0.20.0"
 
+Serializable = Union[int, str, List, Dict]
+
 
 # interface to/from
 def to_json(
     path_or_buf,
     obj,
```

Comment:
I think `obj` should be FrameOrSeries, but since it gets passed too pandas.core.generic I was getting errors like

```sh
pandas/core/generic.py:2401: error: Value of type variable "FrameOrSeries" of "to_json" cannot be "NDFrame"
```

Reply:
I think FrameOrSeries in _typing should be changed to `FrameOrSeries = TypeVar("FrameOrSeries", bound="NDFrame")` as used in #27646